WatchFaceFormat/Weather/watchface/src/main/res/raw/watchface.xml

```diff
@@ -74,15 +74,11 @@
                 </Group>
                 <Group height="47" name="CurrentConditionIcon" width="47" x="215" y="0">
                   <Condition>
-                    <!--
-                      When using CONDITION, it is a requirement that all
-                      potential values are correctly handled. See:
-                      https://developer.android.com/docs/quality-guidelines/wear-app-quality
-                    -->
                     <Expressions>
+                      <Expression name="is_unknown">[WEATHER.CONDITION] == 0</Expression>
```

Comment:
What about > 22 also?

Reply:
It is not enforced, but you shouldn't get values > 22. If you get, it's a problem with the weather provider and in this particular example, nothing would be rendered. A developer might add a `Default` node at the same level with the `Compare` ones, but that is not needed. Should we add a comment about this?

Reply:
I think there's the potential to overcomplicate the sample: The provider shouldn't provide > 22, I think the sample should keep it simple and assume it will do the right thing, which is generally reasonable?

Reply:
We are literally doing this right now.  So the values changing does happen.

I'd consider it maybe more like an exhaustive else.

But feel free to resolve.

Reply:
Default case added